src/VirtualClient/VirtualClient.Main/profiles/PERF-CPU-WRATHMARK.json

```diff
@@ -0,0 +1,18 @@
+{
+  "Description": "Wrathmark CPU Performance Workload",
+  "Metadata": {
+    "RecommendedMinimumExecutionTime": "01:00:00",
+    "SupportedPlatforms": "win-x64",
+    "SupportedOperatingSystems": "Windows"
+  },
+  "Parameters": {},
+  "Actions": [
```

Comment:
We have dependencies like GitClone and DotNetInstallation, check PERF-ASPNETBENCH.json

Reply:
Note it works on Linux and ARM.  It's highly portable.

Reply:
Note that the managed version of the code doesn't put the boards per second in parens for no particular reason other than I didn't quite copy the format string exactly.

Reply:
Note also for the benchmark I used the starting position in endgame.txt

wrath xlt endgame.txt

Is the test  (load the board and make one move).

Other tests are possible but that's what I did.

Reply:
> Note it works on Linux and ARM. It's highly portable. That was basically the point. It can even port down to CLR 4.2.

Thanks. I've included parameters so we can update the tooling and runtime without needing to make code changes.

Reply:
> Note also for the benchmark I used the starting position in endgame.txt
> 
> wrath xlt endgame.txt
> 
> Is the test (load the board and make one move).
> 
> Other tests are possible but that's what I did.

Okay, I will update to use that scenario by default

Reply:
Be sure to use the std-arrays variant.  The main wrath-sharp directory is the Vector128 variant which proved to be a terrible idea even though I did it first.  For bonus points do the inline arrays variant on .NET 8 as an additional case (that's what I did).    Using Vector128 as a fancy inline array ends up being a very bad idea.

Reply:
The standard arrays variant is now the default. The value is a parameter which can be specified on the command line.

`VirtualClient.exe --profile=PERF-CPU-WRATHMARK.json`

`VirtualClient.exe --profile=PERF-CPU-WRATHMARK.json --Subfolder=wrath-sharp-inline-arrays`

The parameterization allows for new variants to be added in the future without requiring pull requests.